include/lsst/afw/image/ImageBase.h

```diff
@@ -249,8 +249,9 @@ class ImageBase {
      * declare this function private
      */
     ImageBase& operator=(const ImageBase& rhs);
-    ImageBase& operator=(ImageBase&& rhs);
-    /// Set the %image's pixels to rhs
+    ImageBase& operator=(ImageBase&& rhs) noexcept;
+
+    virtual /// Set the %image's pixels to rhs
     ImageBase& operator=(const PixelT rhs);
```

Comment:
This looks corrupted. Shouldn't it be:
```suggestion
    /// Set the %image's pixels to rhs
    virtual ImageBase& operator=(const PixelT rhs);
```